apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -253,6 +263,29 @@ router.post(
         }
       });
       return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
+    } else if (req.body.__action === 'send_grades') {
+      // Should something like this be in a model?
+      const assessment = await queryRow(
+        sql.select_assessment_with_course_instance,
+        {
+          assessment_id: req.body.unsafe_assessment_id,
+          course_instance_id: res.locals.course_instance.id,
+        },
+        AssessmentSchema,
+      );
+      if (assessment === null) {
+        throw new Error('Invalid assessment.id');
+      }
+
+      await enrollUsersFromLti13(instance);
+      await updateLti13Scores(assessment.id);
+
+      await queryAsync(sql.update_lti13_assessment_last_activity, {
+        assessment_id: assessment.id,
+      });
+
+      flash('notice', `Sending grades in the background for ${assessment.title}`);
```

Comment:
This is very much not true; your above code will have synced everything by the time we get here. As my previous comment said, we'll want to do this in a server job anyways, and I'd be inclined to just redirect the user to the job page so they can monitor progress.